packages/discovery-provider/scripts/dev-server.sh

```diff
@@ -4,6 +4,10 @@ set -e
 # Audius Discovery Provider / Flask
 # Exports environment variables necessary for Flask app
 
+# Flask uses presence of a .env to determine current directory if left unset
+# https://github.com/pallets/flask/issues/3209
+export FLASK_SKIP_DOTENV=1
```

Comment:
This was so annoying, I was so confused!

Reply:
Thanks python